test/fixtures/dialects/exasol/AlterTableDistributePartition.yml

```diff
@@ -62,10 +62,10 @@ file:
         - comma: ','
         - column_reference:
             naked_identifier: DISTRIBUTE
-        - raw: BY
-        - raw: shop_id
+        - code: BY
```

Comment:
This doesn't look more wrong than it was, but this does look wonky

Reply:
Agreed. But totally separate to this change 😄